interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -226,6 +227,13 @@ namespace cling {
     Preprocessor& PP = m_CI->getPreprocessor();
     DiagnosticsEngine& Diags = m_CI->getSema().getDiagnostics();
 
+    ASTReader* Reader = m_CI->getModuleManager().get();
+    assert(dynamic_cast< ASTDeserializationListener*>(m_Consumer));
+    ASTDeserializationListener* Listener = (ASTDeserializationListener*)m_Consumer;
```

Comment:
This becomes: `ASTDeserializationListener* Listener = cast<ASTDeserializationListener>(m_Consumer);`